ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/BGP_FROM_NETWORK_SERVICES.yml

```diff
@@ -0,0 +1,53 @@
+---
+# bgp-from-network-services-1 should get BGP configured for VRF default since there are BGP peers for the device under the VRF.
+# Notice that the node type l3spine does not have a default overlay or underlay routing protocol, so they will not contribute BGP.
+
+# bgp-from-network-services-2 will *NOT* get BGP configured because of a bug. See issue #3804
```

Comment:
So we need another PR to fix 3804 after this right?

Reply:
correct